### DISCUSSION TITLE
Is/Will there be a way to add persons manually?

### DISCUSSION THREAD

**Discussion by zockol** [nextcloud/recognize#1013] [Q&A] [answered]
Hello, i got a lot of pictures that are clearly a selfy or close to it atleast but faces are not recognized in all of them or even just a few but not all. Will there be a way to manually add those faces in future or just adding the Person X tag to the image so it will appear in someones folder ?

**Comment by aaronm120** [comment]
Definitely needs to have this capability added, but would it be from Recognize or the Memories app to have the interface added?

**Reply by zockol** [reply]
i could imagine that memories app could add an option to add a picture to an person or recognize to manually add a square where the face is, which would be a lot more difficult i guess

**Comment by marcelklehr** [comment, marked as answer]
Hi there,

currently this is not planned (see #779), but might get added in the future. Technology-wise we can currently only make available face detections that were detected by the model, but the capability to add manual detections could be added. These would be hard to make use of in improving automatic assignment of faces, but at least users would have the ability to add missing faces.
